artifacts/deploy/webhook-configuration.yaml

```diff
@@ -209,3 +209,17 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: [ "v1" ]
     timeoutSeconds: 10
+  - name: cronfederatedhpa.karmada.io
+    rules:
+      - operations: ["CREATE", "UPDATE"]
+        apiGroups: ["autoscaling.karmada.io"]
+        apiVersions: ["*"]
+        resources: ["cronfederatedhpas"]
+        scope: "Namespaced"
+    clientConfig:
+      url: https://karmada-webhook.karmada-system.svc:443/validate-cronfederatedhpa
+      caBundle: {{caBundle}}
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions: [ "v1" ]
+    timeoutSeconds: 10
```

Comment:
Please remember to add the configuration for `karmada init`:
https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/cmdinit/karmada/webhook_configuration.go

Can you help to open an issue to track all tasks of this feature?

Reply:
Added https://github.com/karmada-io/karmada/issues/3746